test/OpenSpaceToolkit/Physics/Coordinate/Frame/Providers/TEME.test.cpp

```diff
@@ -91,7 +92,13 @@ TEST (OpenSpaceToolkit_Physics_Coordinate_Frame_Providers_TEME, GetTransformAt)
                 // << " : [TEME - GCRF]" << referenceOrientation_GCRF_TEME.toString(15) << " " << transform_TEME_GCRF.getOrientation().toString(15)
                 // << " >> " << RotationVector::Quaternion(dq_GCRF).getAngle().inArcseconds().toString(3) << " [asec]" << std::endl ;
 
-                // [TBI] Add angular velocity test
+                // Angular velocity test
+
+                const Vector3d referenceAngularVelocity_TEME_ITRF_in_TEME = { referenceRow[5].accessReal(), referenceRow[6].accessReal(), referenceRow[7].accessReal() } ;
+
+                const Vector3d w_TEME_ITRF_in_TEME = transform_TEME_ITRF.accessAngularVelocity() ;
+
+                EXPECT_GT(1e-9, (w_TEME_ITRF_in_TEME - referenceAngularVelocity_TEME_ITRF_in_TEME).norm()) ;
```

Comment:
IMO, a cleaner approach would be to:

1. Define a scenario-bound angular velocity threshold at `const Array<Pair<File, Angle, Real>> referenceScenarios = {...}`
2. Use that scenario-bound threshold here:
```
EXPECT_GT(dw, (w_TEME_ITRF_in_TEME - referenceAngularVelocity_TEME_ITRF_in_TEME).norm()) ;
```

Reply:
Fixed